great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -59,7 +61,12 @@ class _FilePathDataAsset(DataAsset):
     }
 
     # General file-path DataAsset pertaining attributes.
-    batching_regex: Pattern
+    batching_regex: Pattern = MATCH_ALL_PATTERN
```

Comment:
Need this default here because before this default was being set in the manually defined `add_<ASSET_TYPE>_asset()` methods